extensions/cli/geoserver/src/main/java/mil/nga/giat/geowave/cli/geoserver/GeoServerConfig.java

```diff
@@ -96,11 +110,10 @@ public GeoServerConfig(
 	 * Secondary no-arg constructor for direct-access testing
 	 */
 	public GeoServerConfig() {
-		this.propFile = null;
-		this.user = DEFAULT_USER;
-		this.pass = DEFAULT_PASS;
-		this.url = DEFAULT_URL;
-		this.workspace = DEFAULT_WORKSPACE;
+		this(
+				new File(
+						System.getProperty("user.home") + File.separator + ConfigOptions.GEOWAVE_CACHE_PATH
+								+ File.separator + ConfigOptions.GEOWAVE_CACHE_FILE));
 	}
 
 	public String getUrl() {
```

Comment:
now that we are checking for https within the client, it seems like it may make sense to use `https://` on line 124 if we detect a port that ends with `443`